webapp/packages/core-utils/src/throttle.ts

```diff
@@ -41,6 +42,8 @@ export function throttle<T extends (...args: any[]) => void | Promise<void>>(f:
         }
       }, delay);
     }
+
+    return result;
```

Comment:
currently is better to not return any result here because we can't guarantee that we will return it (when throttled we just returning undefined)

so actually type of throttle is wrong it's return synchronous function that will return undefined all time

so we need to change function type OR return proper Promise that will be resolved or rejected